routes/user.js

```diff
@@ -62,34 +86,53 @@ router.route("/")
             });
             res.status(201).send(result);
         }).catch(function (err) {
+
             res.status(406).send("Account with identical email already exists");
         }).done();
     });
 
 // Specific user route
 router.route("/:id")
+    /**
+     * Gets the user and stores it in res.locals.user
+     * @param id of the user that client wants
```

Comment:
Suddenly JavaDoc used :D 
We could use it everywhere but I'm fine with the current usage. Might be nice to mention where that parameter is stored for accessing